quickstart.md

```diff
@@ -172,6 +187,69 @@ Now go back to the k8s node and ensure that you still don’t have any bitnami/n
 root@cluster01-master-0:/home/ubuntu# crictl  -r  unix:///run/containerd/containerd.sock image ls | grep bitnami/nginx
 ```
 
+## Creating a sample Coco workload using enclave-cc
+
+Following the previous example that used the `kata` runtime class, we setup a sample *hello world*
+workload using the `enclave-cc` runtime class for process based TEEs. The deployment below assumes
+the simulated SGX mode build (`ccruntime-enclave-cc-sim.yaml`) is installed by the operator. With that,
+the workload can be deployed on a non-TEE system too.
+
+The example uses a trivial hello world C application:
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: enclave-cc-pod-sim
+spec:
+  containers:
+  - image: docker.io/huaijin20191223/scratch-base:v1.8
+    name: hello-world
+    workingDir: "/run/rune/boot_instance/"
+    command:
+    - /run/rune/boot_instance/build/bin/occlum-run
+    - /bin/hello_world
+  runtimeClassName: enclave-cc
+```
+
+Again, create a pod YAML file as previously described (this time we named it `enclave-cc-pod-sim.yaml`) .
+
+Create the workload:
+```
+kubectl apply -f enclave-cc-pod-sim.yaml
+```
+Output:
+```
+pod/enclave-cc-pod-sim created
+```
+
+Ensure the pod was created successfully (in running state):
+```
+kubectl get pods
+```
+Output:
+```
+NAME                 READY   STATUS    RESTARTS   AGE
+enclave-cc-pod-sim   1/1     Running   0          22s
+```
+
+Check the pod is running as expected:
+```
+kubectl logs enclave-cc-pod-sim | head -5
+```
+Output:
+```
+["init"]
+Hello world!
+
+Hello world!
+
+```
+
+We can also verify the host does not have the image for others to use:
+```
+root@cluster01-master-0:/home/ubuntu# crictl -r unix:///run/containerd/containerd.sock image ls | grep scratch-base
+```
```

Comment:
I'd just add a note saying that users can use the hw version of the payload, in case they have access to hardware that supports SGX.

Reply:
added a note about this